microsoft-edge/toc.yml

```
@@ -496,11 +496,11 @@
 # Performance tool ------------------------------------------------------------
             - name: Performance tool
               items:
-                - name: Get started analyzing runtime performance
+                - name: Introduction to the Performance tool
                   href: devtools-guide-chromium/evaluate-performance/index.md
                   displayName: Performance tool
```

```suggestion
                  displayName: Performance tool, Get started analyzing runtime performance  # old title
```
make this change if ppl might look up old title in upper left multipurpose text box